python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -745,6 +746,26 @@ def resolve_latestMaterializationByPartition(
             for event in ordered_materializations
         ]
 
+    def resolve_latestRunForPartition(
+        self,
+        graphene_info: ResolveInfo,
+        partition: str,
+    ):
+        event_records = list(
```

Comment:
we could optimize this by not fetching the whole event record etc, but I think since this is a query for 1 run for 1 partition when a user clicks that partition we don't need to worry too much